src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -118,10 +118,6 @@ public class StorageAttachedIndex implements Index
     "The large number of tokens can also have a negative impact in performance. " +
     "In most cases it's better to use a simpler query_analyzer such as the standard one.";
 
-    public static final String HINTS_EXCEED_INTERSECTION_CLAUSE_LIMIT_ERROR =
```

Comment:
I am wondering if it will be overengineering to have a warning for users in case the query is using more indexes than the intersection allows in general because of the hints?

Reply:
Yeah, but I was thinking that an unexperienced user may add the hints and the intersection limit is there for a reason :D